CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Workspace lockfile generation for cargo, npm, yarn, and pnpm
 - Go lockfile generation
 - Ignored manifests with a different ecosystem's lockfile in a parent directory
+- Pip requirements.txt parser failing with third-party registries
```

Comment:
Oops...this entry should go in the "Unreleased" section. I can change that when the release is made if time is of the essence.